shared/login/register/error/index.render.desktop.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+import React from 'react'
+import {Text} from '../../../common-adapters'
+import Container from '../../forms/container.desktop'
+import type {Props} from './index.render'
+import {errorMap} from '../../../engine/errors'
+import openURL from '../../../util/open-url'
+
+const renderError = error => {
+  switch (error.code) {
+    case errorMap['scdevicenoprovision']:
```

Comment:
this will change soon. it matches what the CLI does but we want that to change too